__tests__/app.test.js

```diff
@@ -304,4 +304,20 @@ describe('DELETE /api/comments/:comment_id', () => {
                 expect(body.msg).toBe('Invalid comment ID');
             });
     });
+});
+describe('GET /api/users', () => {
+    test('200: responds with an array of users with the correct properties', () => {
+        return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then(({ body }) => {
+                expect(Array.isArray(body)).toBe(true);
+                expect(body.length).toBeGreaterThan(0);
+                body.forEach(user => {
+                    expect(user).toHaveProperty('username');
+                    expect(user).toHaveProperty('name');
+                    expect(user).toHaveProperty('avatar_url');
+                });
```

Comment:
body should not be itself the array of users. The body should be an object with a key of users which holds the array of users